genie-web/src/test/java/com/netflix/genie/web/configs/PropertyBindingIntegrationTests.java

```diff
@@ -38,14 +39,15 @@
  * @author mprimi
  * @since 3.1.0
  */
+@Ignore("This seems more like a test of Spring functionality than Genie functionality, do we need it?")
```

Comment:
To leave it disabled we might as well delete it...

Reply:
Yeah I was going to delete it but I didn't want to do so without giving you a chance to comment on why it existed and if we should keep it around